lambify/bin/_setup

```diff
@@ -1,6 +1,13 @@
 #!/bin/sh
 set -e
 
+if [ "$CI" != "true" ]; then
+  echo "== Cleaning possible prod dependencies from local deploy. =="
+  rm -rf ./.bundle \
+         ./vendor/bundle \
+         ./node_modules
+fi
+
```

Comment:
This is the inverse of what we did in `./bin/_build` file. After someone does a deploy from their own machine. They are now required to run `bin/setup` again. They will likely have `pubic/assets` to remove as well.